packages/notifi-react-card/lib/components/subscription/subscription-card-views/preview-panel/UserInfoPanel.tsx

```diff
@@ -148,15 +154,51 @@ export const UserInfoPanel: React.FC<UserInfoPanelProps> = ({
           ) : null}
         </div>
       ) : null}
-      <button
-        className={clsx(
-          'NotifiPreviewCard__editButton',
-          classNames?.EditButton,
-        )}
-        onClick={handleEditClick}
-      >
-        Edit
-      </button>
+      {contactInfo?.discord?.active ? (
```

Comment:
contactInfo itself isn't nullable is it?

let's update to `contactInfo.discord?.active`

Reply:
addressed